applications/nrf_desktop/doc/power_manager.rst

```diff
@@ -1,7 +1,132 @@
+|                                  |                        |                                  |                      | :ref:`battery_charger`             |
++----------------------------------+------------------------+                                  +                      +------------------------------------+
+|                                  |                        |                                  |                      | :ref:`leds`                        |
++----------------------------------+------------------------+                                  +                      +------------------------------------+
+|                                  |                        |                                  |                      | :ref:`board`                       |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| :ref:`nrf_desktop_main`          | ``module_state_event`` |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| :ref:`ble_state`                 | ``ble_peer_event``     |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| :ref:`usb_state`                 | ``usb_state_event``    |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| :ref:`hid_state`                 | ``hid_report_event``   |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| :ref:`nrf_desktop_power_manager` | ``power_down_event``   |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+|                                  | ``wake_up_event``      |                                  |                      |                                    |
++----------------------------------+------------------------+----------------------------------+----------------------+------------------------------------+
+
+
+External dependencies
```

Comment:
I see this section only in this module?

I would move this whole section to Implementation details or Configuration. After all, these are details about how this module is implemented in nRF Desktop, and partially introduce what you need to select (CONFIG_DESKTOP_POWER_MANAGER_ENABLE) and configure...

Reply:
This is copy paste from watchdog which I used as a base. Will move it and do the same for the watchdog.

Reply:
Done